src/i18n/cs.toml

```diff
@@ -376,3 +379,17 @@ messagePasswordConfirmNotMatch = "Hesla se neshodují"
 messageResponsibilityRequired = "Musíte souhlasit se zajištěním startovného."
 messageTermsRequired = "Musíte souhlasit se zpracováním osobních údajů"
 hintPassword = "Musí obsahovat alespoň 6 znaků a alespoň 1 písmeno."
+
+[routes]
+actionInputDistance = "Zadat vzdálenost"
+labelDistance = "Trasa"
+labelDirectionFromWork = "Z práce"
+labelDirectionToWork = "Do práce"
+labelTransportType = "Typ dopravy"
+
+[routes.transport]
+bike = "Na kole"
+car = "Autem"
+none = "Nijak"
```

Comment:
Small note maybe better translation would be _žádný_.

Reply:
Good point. Added "žádná" as in "žádná cesta".